Mathlib/Algebra/Quaternion.lean

```diff
@@ -293,50 +304,56 @@ variable [Ring R]
 /-- Multiplication is given by
 
 * `1 * x = x * 1 = x`;
-* `i * i = c₁`;
-* `j * j = c₂`;
-* `i * j = k`, `j * i = -k`;
-* `k * k = -c₁ * c₂`;
-* `i * k = c₁ * j`, `k * i = -c₁ * j`;
-* `j * k = -c₂ * i`, `k * j = c₂ * i`. -/
-instance : Mul ℍ[R,c₁,c₂] :=
+* `i * i = c₁ + c₂ * i`;
+* `j * j = c₃`;
+* `i * j = k`, `j * i = c₂ * j - k`;
+* `k * k = - c₁ * c₃`;
+* `i * k = c₁ * j + c₂ * k`, `k * i = -c₁ * j`;
+* `j * k = c₂ * c₃ - c₃ * i`, `k * j = c₂ * i`. -/
```

Comment:
If `k * j = c₂ * i` was true before, it can't be true now, because the meaning of c_2 has changed.